lib/resque/failure/rollbar.rb

```diff
@@ -5,7 +5,21 @@ module Resque
   module Failure
     class Rollbar < Base
       def save
-        ::Rollbar.error(exception, payload)
+        if use_exception_level_filters?
+          payload_with_options = payload.merge(:use_exception_level_filters => true)
+        else
+          payload_with_options = payload
+        end
+
+        ::Rollbar.error(exception, payload_with_options)
+      end
+
+      def use_exception_level_filters?
+        rollbar_version.split('.')[1].to_i > 3
```

Comment:
@dimko exactly, the idea is that we have change the behaviour for `2.3.0` but we don't want to change the behaviour for previous versions.